README.md

```diff
@@ -75,18 +75,16 @@ OLED Pins -> ESP8266 Pins
 To load the firmware onto the ESP8266 I used Platformio.
 
   * Install the drivers for your ESP8266/NodeMCU board.
-  * Install platformio
-  * Install the correct board for platformio to know what to build against
+  * Install [PlatformIO IDE](http://platformio.org/platformio-ide)
+  * Clone this repository or download ZIP archive (unpack it)
+  * PlatformIO IDE `Menu: PlatformIO > Open Project folder`, where specify path to folder that contains `platformio.ini`.
+  * Install the correct board for PlatformIO (`Menu: PlatformIO > Initialize or Update PlatformIO Project`) to know what to build against
```

Comment:
The PlatformIO CLI is good, but PlatformIO IDE is better and works "from a box". Should we add CLI steps?